backend/src/script.ts

```diff
@@ -0,0 +1,54 @@
+import express, { Express } from 'express';
+import cors from 'cors';
+import { db } from './firebase-config';
+import reviewData from './landlord_reviews.json';
+import { Review } from '../../common/types/db-types';
+
+const reviewCollection = db.collection('reviews');
+
+const app: Express = express();
```

Comment:
There's no need to initialize an Express server for the script, since we don't need to call any endpoints or have any need for middleware